palantir-java-format/src/main/java/com/palantir/javaformat/OpenOp.java

```diff
@@ -69,7 +79,7 @@ public static Op make(
 
     @Override
     public void add(DocBuilder builder) {
-        builder.open(plusIndent(), breakBehaviour(), breakabilityIfLastLevel(), debugName());
+        builder.open(this);
```

Comment:
I've also taken the opportunity of passing through OpenOp right into Level rather than unpacking its properties through multiple methods, as it was getting laborious to add new fields.